doc/source/changes/version_0_32.rst.inc

```diff
@@ -58,3 +58,6 @@ Fixes
 
 * fixed the :py:obj:`Array.reindex()` printing a spurious warning message when the
   `axes_to_reindex` argument is the name of the axis to reindex (closes :issue:`812`)
+
+* fixed reading/exporting sessions containing two or more axes/groups
```

Comment:
from the code change below (which is surprisingly small), it seems you only changed the reading part, so I suppose the exporting part was fine, right?

Reply:
Yes. For some reason I don't remember, the name of the loaded axis was set equal to the corresponding hdf key.